docs/development/release.md

```diff
@@ -6,13 +6,59 @@ We use a global versioning approach, meaning version numbers in cumulus are cons
 
 Read more about the semantic versioning [here](https://docs.npmjs.com/getting-started/semantic-versioning).
 
+## Pre-release testing
+
+Before releasing a new major version of Cumulus that contains breaking changes, we should test the deployment upgrade from the latest stable release of Cumulus to the upcoming release with breaking changes.
+
+It is preferable to use the [`cumulus-template-deploy`](`https://github.com/nasa/cumulus-template-deploy`) repo for testing the deployment, since this is the officially recommended deployment configuration for the users.
+
+1. Checkout the `cumulus-template-deploy` repo
+2. Update the deployment code to use the latest stable release artifacts if it hasn't already been updated:
```

Comment:
How does one check if it hasn't already been updated?

Reply:
You have to look in the deployment code to see whether the source for data-persistence and cumulus modules already reference the latest release. Suggestions welcome on how to make this more clear, but I can try to update it

Reply:
Is this better?

35d6eb91364027eb7e5441a4823817672a3d8223